src/components/ui/Navbar/index.tsx

```diff
@@ -7,22 +7,27 @@ import { StyledNavbar } from './Styled';
 import { FlexProps } from '../Flex';
 import { FocusableProps, BaseProps } from '../Base';
 
+
 export interface NavbarProps extends FlexProps, BaseProps, FocusableProps {
   /** Classname to apply custom CSS styles */
   className?: string;
   /** Any react components or HTML elements */
   children?: any;
+  /** optionally render a responsive layout at mobile breakpoints  */
+  responsive?: boolean;
```

Comment:
Should this prop be specific to mobile? In the chance we want to responsiveness at other breakpoints?

Reply:
I'm completely open to a different name for this prop. I don't think that it should be specific to mobile. The use case that brought up this need was using the component in a setting where we *didn't* want the component to display its' styles at the mobile breakpoint. We wanted the component to keep the same layout throughout.